pkg/apis/sessions/session_state.go

```diff
@@ -101,7 +102,13 @@ func (s *SessionState) Age() time.Duration {
 
 // String constructs a summary of the session state
 func (s *SessionState) String() string {
-	o := fmt.Sprintf("Session{email:%s user:%s PreferredUsername:%s", s.Email, s.User, s.PreferredUsername)
+	var o string
+
+	if config.GetMaskSessionPII() {
+		o = fmt.Sprintf("Session{user:%s", s.User)
+	} else {
+		o = fmt.Sprintf("Session{email:%s user:%s PreferredUsername:%s", s.Email, s.User, s.PreferredUsername)
+	}
```

Comment:
I considered having email & preferredUsername hashed, so you could tell when there are identical values, but it feels like the user's id should be sufficient for that use case